src/chainparams.cpp

```diff
@@ -137,7 +137,7 @@ class CMainParams : public CChainParams
         // May, 15 2019 hard fork
         consensus.may2019Height = 582679;
         // Nov, 15 2019 12:00:00 UTC fork activation time
-        consensus.nov2019ActivationTime = 1573819200;
+        consensus.nov2019ActivationTime = nMiningForkTime;
```

Comment:
This is not an improvement IMO. Before, this was a constant. Now it's determined by runtime. I suggest making a constant for `1573819200` (as suggested before) or if not, leave it as it was.

Reply:
Agreed. 

Fixed now (hope so)

```c++
-// FIXME This need to be update every new fork
-uint64_t nMiningForkTime = 1573819200;
+// Next protocol upgrade will be activated once MTP >= Nov 15 12:00:00 UTC 2019
+const uint64_t NOV2019_ACTIVATION_TIME = 1573819200;
+uint64_t nMiningForkTime = NOV2019_ACTIVATION_TIME;
 
 CBlock CreateGenesisBlock(CScript prefix,
     const std::string &comment,
@@ -137,7 +138,7 @@ public:
         // May, 15 2019 hard fork
         consensus.may2019Height = 582679;
         // Nov, 15 2019 12:00:00 UTC fork activation time
-        consensus.nov2019ActivationTime = 1573819200;
+        consensus.nov2019ActivationTime = NOV2019_ACTIVATION_TIME;
```

Reply:
this allows for an emergency abort -- if some last minute issue comes up, people can configure it to sometime in the future and not fork.